### PR TITLE
Update python to 3.6

### LINKED ISSUE #171: Upgrade to python 3.6

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #171 

#### What's this PR do?
Updates python to 3.6

#### How should this be manually tested?
Nothing should break

### CODE REVIEW

File: travis/Dockerfile-travis-watch [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mitodl/bootcamp_watch_travis
+FROM mitodl/bootcamp_watch_travis_next
```
Comment (shaidar) [inline travis/Dockerfile-travis-watch]: @noisecapella What's the `next` for?

Comment (noisecapella) [reply travis/Dockerfile-travis-watch]: We use these uploaded images to skip some of the installation during travis tests

Comment (noisecapella) [reply travis/Dockerfile-travis-watch]: The `next` part lets us have the current image and the future image uploaded at the same time, so this PR can coexist with other PRs

**Comment by shaidar** [comment]
Looks good 👍

**Pull request merged**